api/SVGGraphicsElement.json

```diff
@@ -117,7 +117,7 @@
             "firefox": {
               "version_added": "1.5",
               "notes": [
-                "Before Firefox 128, the `getBBox()` method retured an empty `DOMRect` when there is no fill ([bug 1019326](https://bugzil.la/1019326)).",
+                "Before Firefox 128, the `getBBox()` method returned an empty `DOMRect` when there is no fill ([bug 1019326](https://bugzil.la/1019326)).",
```

Comment:
Nit: Extract these unrelated fixes to a separate PR.

Reply:
No problem 👍🏻

Reply:
done in cb724cd33192e02a83735d34bfcaa9b6bb4367fe